src/web/app/pages-instructor/instructor-course-enroll-page/instructor-course-enroll-page.component.ts

```diff
@@ -177,18 +177,10 @@ export class InstructorCourseEnrollPageComponent implements OnInit {
     this.httpRequestService.post('/course/enrollSave', paramMap, this.enrollData)
         .subscribe((resp: EnrollResultPanelList) => {
           this.showEnrollResults = true;
-          this.statusMessage.pop(); // removes any existing status message
-          this.statusMessage.push({
-            message: 'Enrollment Successful. Summary given below',
-            color: 'success',
-          });
+          this.statusMessageService.showSuccessMessage('Enrollment successful. Summary given below.');
           this.enrollResultPanelList = resp.enrollResultPanelList;
         }, (resp: ErrorMessageOutput) => {
-          this.statusMessage.pop(); // removes any existing status message
-          this.statusMessage.push({
-            message: resp.error.message,
-            color: 'danger',
-          });
+          this.statusMessageService.showErrorMessage(resp.error.message);
```

Comment:
As discussed, let's not do this for the enroll page yet as the output might potentially be a huge chunk of messages.

On that note, can you replicate a huge chunk of error messages and show how it looks like on the snackbar?

Reply:
It looks terrible 🤣 
![image](https://user-images.githubusercontent.com/35655790/55222505-0b76c700-5247-11e9-99b9-b8c06a8f8dd1.png)
Will change it back to the normal status message

Reply:
Besides this one, I'm not sure which page has a long chunk of error message, seems that the ones I see are at most 1-2 lines long (like the screenshot in the PR description).

Reply:
As long as the places where you switched out status messages -> snackbar are tested with the error variant and don't have similar behavior shown above, it should be good :)